integration/integration_test.go

```diff
@@ -462,6 +465,9 @@ func prepare(t testing.TB) {
 		remote = true
 	case "sql":
 		cfg.DatabasePath = "postgres://localhost/cayley_test"
```

Comment:
Can we infer flavor from the protocol part of the URL, if not explicitly stated?